Documentation/Migration/Index.rst

```diff
@@ -0,0 +1,29 @@
+..  include:: ../Includes.rst.txt
+
+..  _migration:
+
+=========
+Migration
+=========
+
+If you want to migrate from using Sphinx to render TYPO3 documentation to
+using the render-guides, the file :file:`Documentation/Settings.cfg` has to be
+replaced by an XML file, :file:`Documentation/guides.xml`.
+
+To facilitate migration the extension `t3docs/typo3-guides-cli` in this
+mono-repository comes with a Symfony Console Command to automatically migrate
+the outdated :file:`Settings.cfg`.
+
+With make
+=========
+
+..  code-block:: sh
+
+    make migrate-settings path=path/to/Documentation
+
+With ddev
+=========
+
+..  code-block:: sh
+
+    ddev exec ddev composer make migrate-settings path=path/to/Documentation
```

Comment:
Remember, later people will require "t3docs/guides-cli" into their extension for the commands. This means, they have to configure a ddev box for their extension, and "build-essential" has to be added manually. So just giving the ddev command won't work for them.

Also this requirement is not mentioned above.

Reply:
This documentation is only for us that use the mono-repository. We can adjust it when we transfer it to the how-to-document to target a wider audience

Reply:
I am using just the plain ol' PHP ;-) But as I am the minority, go ahead.